resources/migrations/001_update_migrations.yaml

```diff
@@ -5152,6 +5152,38 @@ databaseChangeLog:
         - dbms:
             type: mysql,mariadb
 
+  - changeSet:
+      id: v49.2024-01-16T22:32:54
+      author: oisincoveney
+      comment: >-
+        Add report_card.first_published_at
+      changes:
+        - addColumn:
+            tableName: report_card
+            columns:
+              - column:
+                  name: first_published_at
+                  type: ${text.type}
+                  constraints:
+                    nullable: true
+                  remarks: The timestamp when the card was first published in a static embed
+
+  - changeSet:
+        id: v49.2024-01-16T23:27:28
+        author: oisincoveney
+        comment: >-
+          Add report_dashboard.first_published_at
+        changes:
+          - addColumn:
+              tableName: report_dashboard
+              columns:
+                - column:
+                    name: first_published_at
+                    type: ${text.type}
```

Comment:
I want this to be `timestamp.type` but I couldn't get the strings to budge. It might make more sense to handle this completely on the BE